metrics-exporter-prometheus/src/builder.rs

```diff
@@ -194,7 +194,11 @@ impl PrometheusBuilder {
         self
     }
 
-    /// Builds the recorder and exporter and installs them globally.
+    /// Builds the recorder and exporter and installs them globally. Behaves
+    /// differently depending on if called from within a tokio runtime. If
+    /// called within a Runtime, spawns the handler future onto the current
+    /// runtime. If called with no active runtime, spawns a new single-threaded
+    /// tokio runtime and polls the handler future from a new background thread.
```

Comment:
```suggestion
    /// Builds the recorder and exporter and installs them globally.
    ///
    /// If called from within a Tokio runtime, the handler future is spawned directly
    /// into the runtime.  Otherwise, a new single-threaded Tokio runtime is created
    /// on a background thread, and the handler is spawned there.
```